nodeutil/reflect.go

```diff
@@ -631,6 +630,37 @@ func (self Reflect) ReadFieldWithFieldName(fieldName string, m meta.Leafable, pt
 	return v, nil
 }
 
+func MetaNameToFieldNameExt(parent reflect.Value, in string) string {
```

Comment:
Extended... because the original function MetaNameToFieldName() is public, so it cannot be changed. I am open to other suggestions how to hook this is well.

Reply:
how about just MetaNameToField because you are not nec. using field name as you could potentially be using tags

Reply:
Sure... I am not that good with function names, feel free to edit it when you merge.

Reply:
I renamed the function and added a comment to explain what its doing